core/lib/workarea/configuration/sidekiq.rb

```diff
@@ -24,14 +24,6 @@ def load
       end
 
       def configure_workarea!
-        ::Sidekiq::Logging.logger = Logger.new(
-          File.join(Rails.root, 'log', 'sidekiq.log')
-        )
-
-        ::Sidekiq.default_worker_options = {
```

Comment:
Not sure why we had this turned on (it's off by default), with Sidekiq logging to STDOUT, this causes a lot of spam.